### ISSUE TITLE
Game seems to need way too much GPU memory...

### ISSUE THREAD

**Issue by BillGHero** [tsunamayo/Starship-EVO#2997]
### Short Description of the issue:
Game seems to require way too much GPU memory, and will perform very badly if it cannot get it. I seriously doubt the game should require a full 8 Gb of GPU memory just to handle the tutorial area!
...

### Steps to reproduce the issue:
Two machines, one with a GTX 770 (4 Gb) and the other with a GTX 1080 (8Gb). Both machines handle the CPU load ok. But the GTX 770 machine immediately jumps to full GPU memory starting at the main menu and has <= 5 fps in a new world's tutorial ring area. The other machine shows 5Gb used out of 8 at the menu and full 8 Gb used in the same new game tutorial area. The net effect is that the game only performs ok on the machine with the 8Gb GPU...
...

### Environment
- Starship EVO version: 20w36b
- Graphics Card: GTX 770 / GTX 1080

-OS: Manjaro Linux
-Proton: 5.0-9

### Additional information:

Normally I would have just written the issue off as being because I am running it through Proton and not a native Windows platform. And I have not completely ruled that out by any means. However, I would generally expect that the game would have problems irregardless of the 'GPU'/'GPU memory' if that were the case.

Outside of this issue the game looks very promising. Keep up the good work!

**Comment by ProPeach** [comment]
For reference, I'm running an RX580 8GB on straight Windows 10 and I peak at around 6GB VRAM usage, but often see very high (80-100%) GPU usage while playing the game. Seems like it's quite common going off various threads like #2714, and might be related to #2836. Perhaps there's a memory leak somewhere?

**Comment by tsunamayo** [comment]
Hum not sure if it is a memory leak yet, I allocate a % of VRAM for various buffers, but that should be well below the max.
What is your monitor resolution, and is the world blank apart from the starting ships?
Can you monitor your usage over time? Is it right away at full memory?
I code this baby on a GTX960 with 2go RAM, with no issue in the starting area. A few other player on linux are reporting low FPS - so my guess it is more Linux related than VRAM (but that could be VRAM issue  on linux...)

Thanks

**Comment by BillGHero** [comment]
No, this is a starting world with the big ring and tutorial buildings. The game uses 5 Gb of GPU RAM at the menu and all of it in the game world (for the machine with a GTX 1080). The other machine (GTX 770) maxes out GPU RAM at the menu.

I have a little time and I am going to see if I can try different versions of the game and different versions of Proton.

By the way, thank you @tsunamayo checking in on this post; particularly since I am using a non-supported OS :)

**Comment by BillGHero** [comment]
Ok, I can get a little bit of memory back on the big card (1080) by going from 2k resolution to 1080p. But I am also wondering:

@tsunamayo  Does the game try to use the GPU's for calculations too? (OpenCl or equivalent?) I ask this because the machine with only a GTX 770 takes much longer to load the same game world.

**Comment by tsunamayo** [comment]
@BillGHero yes it does (not openCL though, just the DX11 compute shaders). But it will be unplugged for the starting menu. You still need to wait for the galaxy skybox compute though on loading the game...

**Comment by Misterio77** [comment]
I'm having some trouble with high vram usage as well.
Running the game under Proton, with a gtx 970 (4gb vram). I get about 10fps in the tutorial area :c

**Comment by WILDCATreactor** [comment]
this might be why my GPU was having overheat issues as well, 100% usage on the MSI R9 390, also an 8gb card when I was tracking the heat issue. with two extra buildings (literal slabs for landing on) and three vehicles sitting in the starter area.
